modules/price/src/Calculator.php

```diff
@@ -180,6 +180,32 @@ public static function trim($number) {
   }
 
   /**
+   * Rounds the number to specified precision.
+   *
+   * @param string $number
+   *   The number to trim.
+   * @param int $precision
+   *   The number of decimal digits to round to.
+   * @param int $mode
+   *   The rounding mode.
+   *
+   * @return string
+   *   The rounded number.
+   */
+  public static function round($number, $precision = 0, $mode = PHP_ROUND_HALF_UP) {
```

Comment:
Yeah, rounding needs to be used. I didn't finish porting in all options as `bcmath` operations.

@drugan we're only rounding in the order total, because we want the most precision. We don't want to round until the final price is calculated.